### ISSUE TITLE
Request for Document Viewer on ASSETS LIBRARY

### ISSUE THREAD

**Issue by drasticdpk** [appirio-tech/connect-app#3947]
Can we add document viewer functionality on Assets Library Module ? Now user has to download the file to view/access it. 
As a user most of the time I want to view/read the document and I will only download the file if there is anything need to change or share to someone. cc @lakshmiathreya

**Comment by lakshmiathreya** [comment]
@acshields This is a usability issues. Pls review! Cc @vikasrohit

**Comment by vikasrohit** [comment]
I don't think we can build a rich document viewer for every type of documents. If the client machine has the relevant plugins and softwares, browser automatically renders the file e.g. in my case I can easily view the PDFs in my browser in separate tab when clicked on the PDF document in my project.
That is my view, but we can think of exploring some ready made document viewer if available, if this features comes on priority list @acshields.

**Comment by acshields** [comment]
@drasticdpk @vikasrohit - I think this is great feedback, but for now I am not going to prioritize exploring this feature. I think it's a great nice to have, but we have some heavier-hitting features that we need to accomplish first.
